tasks/main.yml

```diff
@@ -51,9 +51,13 @@
     repo: "{{ git_url }}"
     dest: "{{ checkout_dir }}"
     version: "{{ git_version | default( 'HEAD' ) }}"
+    force: yes   # needed because of submodules to avoid "Local modifications exist in repository"
```

Comment:
So I may have tried in the past and might have add issues. But maybe not, let's merge and see.